NtupleTools/test/make_ntuples_cfg.py

```diff
@@ -565,6 +565,25 @@
 # Generate analyzers which build the desired final states.
 final_states = [x.strip() for x in options.channels.split(',')]
 
+
+def order_final_state(state):
```

Comment:
This can be done in one line
```
''.join(sorted('emmtmtmgjjmtmeee', key=lambda x: 'emtgj'.index(x)))
```
though you lose the invalid char check. So add a check at the beginning.
Of course. This is just pedantic, it functions as is.